1_basics.ipynb

```diff
@@ -206,18 +408,40 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "ja"
+   },
    "source": [
-    "## 1. スクリーンショットの埋め込み"
+    "## スクリーンショットの埋め込み"
    ]
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "## Screenshot embedding"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "lang": "ja"
+   },
    "source": [
     "スクリーンショットの取得と埋め込みは以下のコマンドで実行可能です。"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "You can take and embed screenshots with the following command:"
```

Comment:
Please choose the sentence that is most appropriate to the context:

_1 command_: "You can take and embed screenshots with the following **command**:"

_more that one command_: "You can take and embed screenshots with the following **commands**:"